AdvancedApplications/ODBCApplication/README.md

```diff
@@ -1,20 +1,29 @@
-# Instructions on How to Run ODBC Applications And DB2CLI For IBM Db2 Event Store
+# ODBC Remote Application / DB2CLI Execution Steps:
 
-## ODBC Remote Application / DB2CLI Execution Steps:
+Prior to running the sample application you must first set up the environment to run a ODBC/DB2CLI application. You must have completed the following steps found [here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/tree/master/AdvancedApplications#odbcdb2cli-setup). If you are using the Docker container, the instructions are executed inside the container. 
 
-This assumes that you have set up the environment to run a ODBC/DB2CLI application. If you haven't done so, [please follow the instructions here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/tree/master/AdvancedApplications#odbcdb2cli-setup) to set up your environment. 
+If you are not running in the Docker container, run the script to configure SSL which is used when setting up the container which can be found [here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh). The script downloads the SSL server certificate through the REST API. This is necessary as Db2 Event Store is configured with SSL with a default keystore out of the box (this may change if you configure your own keystore after installation). Also the following environment variables must be set up
```

Comment:
> Also the following environment variables must be set up

Add "in order to run the sample application"

Reply:
NIT - "configured with SSL with a default keystore out of the box" ... can you change this to "configured with SSL using a dynamically generated self signed certificate out of the box" ... that is more accurate.

Reply:
changes made